src/data_processing.py

```diff
@@ -0,0 +1,29 @@
+from csv import reader
+
+def read_csv(filename):
+    with open(filename, 'r') as read_obj:
+        csv_reader = reader(read_obj)
+
+        for row in csv_reader:
+            if ("Game" in row[0]):
+                print(row[0])
+
+def combine_list_to_dictionary(row,row2):            
+    stats = {}
+
+    row = ['Player', 'REB', 'AST', 'BLK', 'STL', '2PM', '2PA', '3PM', '3PA', 'TO', 'PTS', 'FGA', 'FG%', '3P%', '2P%']
+    row2 = ['Brian', '3', '1', '', '1', '1', '1', '1', '4', '', '3', '5', '40.00%', '25.00%', '100.00%']
```

Comment:
These two lines are no longer needed. You have `row` and `row2` from the input parameters.

Reply:
Got it. Thank you for letting me know.